### ISSUE TITLE
[Request]: Export other YTDL-core functions

### ISSUE THREAD

**Issue by NightYoshi370** [favware/node-packages#2]
**Is your feature request related to a problem? Please describe.**
For my music bot on Discord, I'd like to have information about the song as well as the player itself

**Describe the ideal solution**
Exporting ytdl-core functions alongside the custom play one

**Describe alternatives you've considered**
Use both ytdl-core and ytdl-prismedia

**Further details:**
- Affected package(s): ytdl-prismedia
- Version of the package: Latest
- Priority this request should have – please be realistic and elaborate if possible:
  - [X] Low
  - [ ] Medium
  - [ ] High
  - [ ] Severe

**Comment by Favna** [comment]
Sure I can do this. No ETA's guaranteed but I should be able to do it by tomorrow.

**Comment by Favna** [comment]
Released in v2.2.0.

You should however note that you still do need the peerDependency of ytdl-core!

Before:
```ts
import { play } from '@favware/ytdl-prismplayer';
import { downloadOptions } from 'ytdl-core';

// or CommonJS:
const { play } = require('@favware/ytdl-prismplayer');
const { downloadOptions } = require('ytdl-core');
```

After:
```ts
import { play, downloadOptions } from '@favware/ytdl-prismplayer';

// or CommonJS:
const { play, downloadOptions } = require('@favware/ytdl-prismplayer');
```
